packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -122,6 +117,10 @@ const AccountTransferForm = observer(
             onChangeConverterToAmount,
             resetConverter,
         } = crypto_fiat_converter;
+        const is_open_order_position_status_present = useIsMt5LoginListStatusPresent(
+            selected_to.value ?? '',
+            'open_order_position_status' // this flag is not yet present in the BE response.
+        );
```

Comment:
I think we can move this `selected_to` from useCashierStore into the `useIsMt5LoginListStatisPresent` hooks instead of passing in as a props 🤔 so that the hook will look cleaner.

Reply:
@suisin-deriv if `selected_to` is moved inside the hook, then it becomes less re-usable. right? i mean, in this scenario, we can pass any account login id, whereas if i use account login id from useCashierStore, only that account id can be used inside the hook. am i missing something? 💭